boards/common/particle-mesh/Makefile.dep

```diff
@@ -6,5 +6,7 @@ ifeq (1,$(PARTICLE_MONOFIRMWARE))
   USEMODULE += usb_board_reset
 endif
 
+include $(RIOTBOARD)/common/makefiles/stdio_cdc_acm.dep.mk
```

Comment:
The particle debugger does have a UART (that also comes up as `/dev/ttyACM0`), so if you want to avoid unnecessary USB support, you could add

```suggestion
ifeq (1,$(PARTICLE_MONOFIRMWARE))
  include $(RIOTBOARD)/common/makefiles/stdio_cdc_acm.dep.mk
endif
```

if that variable is still available here

Reply:
At least according to the schematics I found on https://docs.particle.io/datasheets/discontinued/xenon-datasheet/, the 10-pin header is only wired for SWD, so any UART there is would be about semihosting (which I've found no hint to in the remainer of the board's RIOT port.)